bigtable/client/client_options_test.cc

```diff
@@ -122,6 +123,26 @@ TEST(ClientOptionsTest, SetGrpclbFallbackTimeout) {
   // 2nd element of test_args.
   EXPECT_EQ(GRPC_ARG_GRPCLB_FALLBACK_TIMEOUT_MS,
             grpc::string(test_args.args[1].key));
+
+  // Test seconds are converted into milliseconds
```

Comment:
Please separate these assertions to their own `TEST(ClientOptionsTest, ...)` functions.